src/api/index.js

```diff
@@ -16,8 +24,14 @@ const updateData = (data, resolve, reject) => {
     .then(res => resolve(res))
     .catch(err => reject(err));
 };
+const aboutUser = (data, resolve, reject) => {
```

Comment:
same here

Reply:
I have renamed it from `aboutUser` to `userAboutInfo`